Sources/GPX/OAGPXDatabase.mm

```diff
@@ -215,6 +218,17 @@ -(BOOL)updateGPXItemPointsCount:(NSString *)fileName pointsCount:(int)pointsCoun
     return NO;
 }
 
+-(BOOL)updateGPXFilePath:(NSString *)filePath newFilePath:(NSString *)newFilePath
+{
+    for (OAGPX *item in gpxList) {
```

Comment:
These changes will not be saved in the GPXDatabase and will probably cause inconsistent app behavior.